lib/engine/game/g_1860.rb

```diff
@@ -239,7 +241,7 @@ def close_other_companies!(company)
       end
 
       def biggest_train(corporation)
-        corporation.trains.map(&:distance).max || 0
+        corporation.trains.map { |t| t.distance.first }.max || 0
```

Comment:
there is no such thing as distance first, the format doesn't look correct

Reply:
distance is an array here.

Reply:
Moved to standard train definitions.